server.js

```diff
@@ -24,9 +18,78 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Api endpoints: /shows && /shows/:id && /shows/year/:year && ?type=Movie || ?listed= Dramas')
```

Comment:
[Here](https://www.npmjs.com/package/express-list-endpoints) is a cool lib if you want to include some documentation in your API 🙌